mycroft/client/speech/main.py

```diff
@@ -157,6 +157,7 @@ def main():
     ws.on('recognizer_loop:audio_output_start', handle_audio_start)
     ws.on('recognizer_loop:audio_output_end', handle_audio_end)
     ws.on('mycroft.stop', handle_stop)
+    ws.on('recognizer_loop:reload', loop.reload)
```

Comment:
I think this may fail since the loop.reload method doesn't take a message parameter

Reply:
Oh, nice catch, and this isn't actually needed since it's no longer used in the code implementation. I'll just remove it.